fw/t/unit/test_http_parser_common.h

```diff
@@ -461,12 +463,19 @@ do_split_and_parse(int type, int chunk_mode)
 		req = test_req_alloc(GET_FRAMES_TOTAL_SZ());
 	}
 	else if (type == FUZZ_REQ_H2) {
-		if (req)
-			test_req_free(req);
+		static TfwH2Ctx	h2_origin;
+
+		if (chunk_size_index == 0)
+			h2_origin = conn.h2;
+		else
+			conn.h2 = h2_origin;
```

Comment:
Please write a comment why do we need to save and restore HTTP/2 context for each request processing

Reply:
Done.